packages/nodes-base/nodes/Redis/Redis.node.ts

```diff
@@ -264,7 +274,20 @@ export class Redis implements INodeType {
 				required: true,
 				description: 'The key pattern for the keys to return',
 			},
-
+			{
+				displayName: 'Get Values',
+				name: 'getValues',
+				type: 'boolean',
+				displayOptions: {
+					show: {
+						operation: [
+							'keys',
+						],
+					},
+				},
+				default: true,
```

Comment:
Setting this as default true will be a breaking change

Reply:
This preserves the [current default behaviour](https://github.com/n8n-io/n8n/blob/88dea330b9605eabcafa4288f01f1a68763d0629/packages/nodes-base/nodes/Redis/Redis.node.ts#L599) where the value of the keys are pulled when listed. The reason to provide this toggle is when using lists you can often have millions of items that will get pulled which is infeasible if you just want to know what keys exist.

Reply:
Ahh, my bad. Not sure why I thought it was not the default behavior.